test/verify/check-subscriptions

```diff
@@ -130,8 +131,8 @@ password=foobar
 insecure_connection=True
 """)
 
-        m.upload([ "verify/files/mock-insights" ], "/var/tmp")
-        m.spawn("/var/tmp/mock-insights", "mock-insights")
+            m.upload([ "verify/files/mock-insights" ], "/var/tmp")
+            m.spawn("/var/tmp/mock-insights", "mock-insights")
 
     def testRegister(self):
```

Comment:
`testRegister` will fail as well, won't it? If so, then you can skip the whole `TestSubscriptions` class

Reply:
No, it's supposed to pass (and did). This just covers the actual subscription, not the insights part. I'd really like to keep testing that part, as we ship the cockpit-subscriptions bits in RHEL 7.

Reply:
Oh, my bad, now I see that it failed in setUp, sorry :)